docs/api/createApi.md

```diff
@@ -216,12 +219,36 @@ Defaults to `false`. This setting allows you to control whether RTK Query will o
 - `true` - Will always refetch when a new subscriber to a query is added. Behaves the same as calling the `refetch` callback or passing `forceRefetch: true` in the action creator.
 - `number` - **Value is in seconds**. If a number is provided and there is an existing query in the cache, it will compare the current time vs the last fulfilled timestamp, and only refetch if enough time has elapsed.
 
-If you specify this option as well as with `skip: true`, this **will not be evaluated** until `skip` is false.
+If you specify this option alongside `skip: true`, this **will not be evaluated** until `skip` is false.
 
 :::note
 You can set this globally in `createApi`, but you can also override the default value and have more granular control by passing `refetchOnMountOrArgChange` to each individual hook call or when dispatching the [`initiate`](#initiate) action.
 :::
 
+### `refetchOnFocus`
```

Comment:
Are we missing docs that `setupListeners` is required to use this feature? (And on `setupListeners` in general)